dcr_config.yaml

```diff
@@ -21,3 +21,5 @@ cleanup_interval: "15sec"
 open_blobs_soft_limit: 2
 # Hard limit for open blobs
 open_blobs_hard_limit: 2
+# Degree of parallelism in disks initialization
+init_par_degree: 4
```

Comment:
I think it's better to name it like `init_threads` or smth like that

Reply:
`degree of parallelism`  is the standard naming for such parameters. `threads` is less obvious